<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Earth and Moon</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>planet-earth.ico</normaloff>planet-earth.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="OpenGLWidget" name="moonAndEarthWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>841</width>
          <height>451</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="moonOrbiteLabel">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>Орбита Луны</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="moonOrbiteSlider">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>15</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="equatorAngleLabel">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>Угол наклона относительно экватора</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="equatorSlider">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>15</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="greenwichMeridianLabel">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16</height>
             </size>
            </property>
            <property name="text">
             <string>Угол поворота относительно Гринвича</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="greenwichSlider">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>15</height>
             </size>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::NoTicks</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OpenGLWidget</class>
   <extends>QWidget</extends>
   <header location="global">openglwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>moonOrbiteSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>moonOrbiteSliderMoved(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>487</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>equatorSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>equatorSliderMoved(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>568</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>558</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>greenwichSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>greenwichSliderMoved(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>422</x>
     <y>611</y>
    </hint>
    <hint type="destinationlabel">
     <x>719</x>
     <y>602</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>moonOrbiteSliderMoved(int)</slot>
  <slot>equatorSliderMoved(int)</slot>
  <slot>greenwichSliderMoved(int)</slot>
 </slots>
</ui>
